egs/gop_speechocean762/s5/RESULT

```diff
@@ -0,0 +1,12 @@
+MSE: 0.15
```

Comment:
It  would be better to add documentation for this file, e.g. MSE(mean_sequared_error), and what `0/1/2` means, what `macro/weighted avg` etc.

Reply:
I just updated the documents of both this file and the `README` in the `speechocean762` tar.